### DISCUSSION TITLE
Loading & LoadingOverlay -- customised icon via attribute?

### DISCUSSION THREAD

**Discussion by cstrat** [mantinedev/mantine#83] [Feedback]
While I do love the default loading icon with mantine, it would be great if we could simply pass in an icon to be used.
For `<Loading>` this may not be worth the effort, as you could simply just import your own icon anyway.

However the `<LoadingOverlay>` provides a nice utility with the transparent screen over the parent container.

I'd love to be able to do: `<LoadingOverlay icon={<FontAwesomeIcon icon={faSpinner} spin />} />` and have the overlay effect with my spinner.

This doesn't seem to be something we can manually do with the `<Overlay>` component as it purely an overlay without children.

**Comment by rtivital** [comment]
Good point, I've already added this feature in next minor release, it will accept any react node and display as loader:
<img width="727" alt="Снимок экрана 2021-05-18 в 17 40 46" src="https://user-images.githubusercontent.com/10353856/118671518-35611e00-b800-11eb-8fdf-0d93aa0efed1.png">
